pkg/static_unit/plugin_static_unit.go

```diff
@@ -98,6 +105,45 @@ type staticAssetPathMatcher struct {
 	err         error
 }
 
+func (m *staticAssetPathMatcher) ModifyPathsForAnnotatedFile(path string) error {
```

Comment:
Can you doc a bit what this function does? I'm getting lost in the "Abs", "Rel" etc. It likely is similar to how readdir works, but I've forgotten how that works 😄